src/apply_conf.rs

```diff
@@ -183,29 +183,36 @@ fn copy_connection_files(
 
     for interface in &host.interfaces {
         info!("Processing interface '{}'...", &interface.logical_name);
+        let connections = &interface
+            .connection_ids
+            .clone()
+            .unwrap_or(vec![interface.logical_name.clone()]);
```

Comment:
Is it safer to always include the logical name? Do we anticipate that it will always be present in the `connections_id`?

Reply:
I added the logical_name here as a fallback mechanism mainly for backwards compatibility. Although I doubt if we need backwards compatibility (I assume same NMC version is used for generating and applying).

No, the logical name won't be always present in the `connection_ids`.

See also the example 
```yaml
    - logical_name: ovs0
      connection_ids:
        - ovs0-port
        - ovs0-if
      interface_type: ovs-interface
```

Reply:
Isn't there an `ovs0.nmconnection` file that we don't copy over with the current implementation?

Reply:
`ovs0.nmconnection` is not generated by nmstate. When applying this configuration with the current implementation you'll get an error like described in #113 because the file is missing